delivery_tnt_oca/models/sale_order.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    tnt_last_request = fields.Text(
+        string="Last TNT API request", help="Used for debugging", readonly=True
+    )
+    tnt_last_response = fields.Text(
+        string="Last TNT API response", help="Used for debugging", readonly=True
+    )
```

Comment:
If these fields are only used for debugging, then I think it's more useful to use logs for that. Just remove these and log the data you think might be needed in the future and you're done.